plugins/uefi/fu-plugin-uefi.c

```diff
@@ -742,6 +773,36 @@ fu_plugin_unlock (FuPlugin *plugin, FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_plugin_uefi_create_dummy (FuPlugin *plugin, GError **error)
+{
+	const gchar *key;
+	g_autoptr(FuDevice) dev = fu_device_new ();
+
+	key = fu_plugin_get_dmi_value (plugin, FU_HWIDS_KEY_MANUFACTURER);
+	fu_device_set_vendor (dev, key);
```

Comment:
do we not need to check for NULL here?

Reply:
Yeah, good catch.  I adjusted both cases it was pulling from DMI table.